dependencies.gradle

```diff
@@ -21,13 +20,13 @@ ext.dep = [
   "grpcNetty": "io.grpc:grpc-netty:1.21.0",
   "grpcProtobuf": "io.grpc:grpc-protobuf:1.21.0",
   "grpcStub": "io.grpc:grpc-stub:1.21.0",
-  "guava": "com.google.guava:guava:30.0-jre",
-  "guavaTestLib": "com.google.guava:guava-testlib:30.0-jre",
+  "guava": "com.google.guava:guava:29.0-jre",
```

Comment:
Downgrade?

Reply:
Good catch - I was just using the dep-upgrade and reverting deps that caused build problems.  I'll revert these and retry the build.